openrelik_worker_common/archives.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Helper methods for archives."""
+import os
+import subprocess
+import time
+from uuid import uuid4
+
+
+def unpack(input_path: str, output_folder: str, log_file: str) -> str:
```

Comment:
We will have support for more archive types, rename to reflect the archive type
```suggestion
def extract_7zip(input_path: str, output_folder: str, log_file: str) -> str:
```

Reply:
You right, I'm just used to use separate tools for the archives :) Still good to name it descriptive IMO 👍🏼